### DISCUSSION TITLE
Multilevel categories without parent selection

### DISCUSSION THREAD

**Discussion by alexanderbuergin** [craftcms/cms#9769] [Ideas] [answered]
By organizing categories over levels Craft selects all parents by default, but in many cases you only like to use the child of an entry. 

It could be very helpful to add the option to select if the category or the field should behave like this or not – so that the parents don't are included to the selection.

**Comment by brandonkelly** [comment, marked as answer]
For now your best bet would be to use entries instead of categories for this. In the long run we plan to merge the concepts of categories and entries together anyway (#9781).
